src/frontends/onnx/frontend/include/openvino/frontend/onnx/frontend.hpp

```diff
@@ -36,6 +29,13 @@ class ONNX_FRONTEND_API FrontEnd : public ov::frontend::FrontEnd {
 protected:
     InputModel::Ptr load_impl(const std::vector<ov::Any>& params) const override;
 
+    // m_extensions should be the first member here,
+    // m_extensions can contain SO Extension (holder for other Extensions),
+    // so it should be released last.
+    std::vector<Extension::Ptr> m_other_extensions;
+    std::vector<DecoderTransformationExtension::Ptr> m_transformation_extensions;
+    std::vector<ConversionExtensionBase::Ptr> m_conversion_extensions;
+
 private:
     ExtensionHolder m_extensions;
```

Comment:
why do we need so many fields with extensions?

Reply:
`ExtensionHolder` is more convenient way to pass extensions (Telemetry, ProgressBar) which should collect information from different parts of FrontEnd to necessary classes (InputModel, Graph and so on). It's just a structure to combine extension of this type.
`m_transformation_extensions `and `m_conversion_extensions `are needed to avoid searching and converting these extensions when using them in FrontEnd class.
`m_other_extensions `stores `.so` extensions for now